querier_test.go

```diff
@@ -229,6 +229,7 @@ func createIdxChkReaders(tc []struct {
 
 	postings := &memPostings{m: make(map[term][]uint32, 512)}
 	chkReader := mockChunkReader(make(map[uint64]chunks.Chunk))
+	lblIdx := make(map[string]map[string]bool)
```

Comment:
Can we use this here: https://github.com/prometheus/tsdb/blob/3a36551b907faa38e7eddfa87da9826ad90cd1cb/postings.go#L321-L343